experimental/Schemes/src/Resolution_structure.jl

```diff
@@ -570,53 +742,83 @@ function desingularization(X::AbsAffineScheme; algorithm::Symbol=:BEV)
   return desingularization(CoveredScheme(X); algorithm)
 end
 
+function weak_to_strong_desingularization_surface(phi::BlowUpSequence)
+  return weak_to_strong_desingularization_surface(mixed_blow_up_sequence(phi))
+end
+
+function weak_to_strong_desingularization_surface(phi::MixedBlowUpSequence)
+  dim(domain(phi)) == 2 || error("not implmemented yet")
+
+  !phi.is_strong || return phi
+  ex_divs = phi.ex_div
+  for i in normalization_steps(phi)
+    !is_one(ex_divs[i]) || continue
```

Comment:
Personally I find these negated statements combined with `||` harder to understand compared to the un-negated statement combined with `&&`. So like this:
```suggestion
    is_one(ex_divs[i]) && continue
```
Similar in other places, but I won't bother you with those, decide for yourself what you prefer.

Reply:
This is personal preference. Sorry. For me it reads as a kind of assertion for the respective loop. I do not follow your remark here.